lib/test-handler/helpers/describe.js

```diff
@@ -0,0 +1,99 @@
+const { api } = require('../../api');
+const { config } = require('../../config/config');
+const {
+    reportException,
+    getTestKey,
+    formatStatus,
+    getRelativePath,
+    getStatus,
+    formingSteps,
+} = require('./common');
+
+/**
+ * Separate test key to key and version.
+ *
+ * @param {string} testKey - Test key.
+ * @returns {{version: string, key: string}} Separated key and version.
+ */
+const getTestKeyVersion = testKey => {
+    const result = RegExp(/(.*?)(?:\((.*)\))/).exec(testKey);
+    return {
+        key: (result[1] || '').trim(),
+        version: (result[2] || '').trim(),
+    };
+};
+
+/**
+ * Formats data test suites.
+ *
+ * @param {Object} test - Test.
+ * @returns {String} Result of the test suite.
+ */
+const getTestName = test => test.testResults[0].ancestorTitles[0];
+
+/**
+ * Get script errors.
+ *
+ * @param {Array} testResult - Result of the test case.
+ * @returns {Object} Script errors for JTM.
+ */
+const getScriptErrors = testResult => testResult.map(testCase => testCase.comment).join('\n');
+
+/**
+ * Form whitelist according to query in config file.
+ *
+ * @returns {Promise<Object>} - Key-version map.
+ */
+const getTestCaseWhitelist = async () =>
+    (await api.testCase.search(config.whitelist.query || '')).reduce(
+        (whitelist, testCase) => ({ ...whitelist, [testCase.key]: testCase.majorVersion }),
+        {},
+    );
+
+/**
+ * Format result for Test Cycles.
+ *
```

Comment:
Describe please, from which and to which this function does format

Reply:
it works the same as for case strategy, but also marks tests as blocked if they are not in whitelist